Helper/Data.php

```diff
@@ -1326,15 +1326,11 @@ public function getOpenInvoiceLineData(
         $itemVatPercentage,
         $numberOfItems,
         $payment,
-        $itemId = null
+        $itemId
```

Comment:
Forcing an optional argument to required could be breaking change. Let's discuss it.